modules/ROOT/pages/pricing.adoc

```diff
@@ -2,86 +2,50 @@
-
-== Supported Mule Runtime Versions
-
-To benefit from the robust usage reports that accompany usage-based pricing plans, all of your applications (in production and sandbox) must use the following Mule versions:
-
-[cols="2*"]
-|===
-|Deployment Type |Version Requirements
-
-|CloudHub 2.0
-|The latest date-patch versions for 4.3 and 4.4
-
-|CloudHub 1.0
-|The latest date-patch versions for 3.9, 4.2, 4.3 and 4.4
-
-|Runtime Fabric
-a| One of the following versions or any later date-patch:
+The Usage Reports page supports pricing packages for integration. When your organization opts in to a usage-based pricing plan, such as the Anypoint Integration Advanced package, your apps must be redeployed. After redeploying your apps, your usage reports are shown. 
 
-Mule 4 images version:
+For CloudHub and CloudHub 2.0, apps are automatically redeployed during the monthly runtime patch update cycle. For more information, see xref:runtime-manager::cloudhub-app-runtime-version-updates.adoc[CloudHub Runtime Continuous Updates] and xref:cloudhub-2::ch2-patch-updates.adoc[CloudHub 2.0 Operating System Patch Updates].
 
-* 4.4.0-20230111-1
-* 4.3.0-20221212-2
-* 4.2.2-20221027-7
-* 4.2.1-20221027-7
-* 4.2.0-20221027-6
-
-Mule 3 images version:
-
-* 3.9.5-20230424-2
-* 3.9.4-20230424-1
-* 3.9.3-20230424-1
-* 3.9.2-20230424-1
-* 3.9.1-20230424-1
-
-|=== 
-
-After your organization onboards to usage-based pricing, you must redeploy your apps with a supported version, either manually or during the monthly patching cycle. The monthly patching cycle is applicable only for CloudHub 1.0 and CloudHub 2.0 applications. RTF customers must be using RTF agent version 1.13.112 or 2.1.30 or newer. You must also manually redeploy Runtime Fabric applications (with a supported date-patch version) after your organization is onboarded to usage-based pricing.
-
-To xref:cloudhub-2::ch2-update-apps.adoc#roll-back-to-a-previous-successful-configuration[roll back] to a previous successful configuration of existing applications that use a Mule version that the usage-based pricing model does not support, choose a compliant version before deploying changes. After deployment, MuleSoft creates a new configuration with the previously chosen settings and a supported runtime.
+For Runtime Fabric, you must redeploy apps manually. You must redeploy Runtime Fabric apps in a timely manner to remain in compliance with your Anypoint Platform contract.
 
 == Metrics Definitions
 
 Usage-based pricing packages and their usage reports are based on your organization's use of allotted Mule components.
 
-For information about pricing concepts within Anypoint Platform, see xref:general::pricing-metrics.adoc[Usage and Pricing Metrics].
+To understand the technical aspects of metering and pricing concepts within Anypoint Platform, see xref:general::usage-metrics.adoc[Usage and Pricing Metrics] documentation.
```

Comment:
Do you need the hard-coded link text and "documentation"? Can you just link to that topic?

Reply:
Nope